.eslintrc

```diff
@@ -92,4 +92,9 @@ rules:
   prefer-arrow-callback: 2
   prefer-const: 2
   prefer-spread: 2
-  prefer-template: 2
\ No newline at end of file
+  prefer-template: 2
+
+overrides:
+- files: "**/*.test.js"
```

Comment:
maybe this has an error, try this https://github.com/asyncapi/generator/blob/b071cebe39beb50806cb2872e1256e7b7a328565/test/renderer.test.js#L1

Reply:
Nope 😞 Still 36% of duplication...